enhancements/sig-architecture/33-detached-deploy-mode/detached-deploy-mode.md

```diff
@@ -0,0 +1,224 @@
+
+```
+                    +-----------------------------------------------------------------+                                 
+                    | mnagement-cluster                                               |                                 
+                    |                                                                 |                                 
+                    | cluster-manager                         klusterlet              |                                 
+                    | +-------------------+                   +-------------------+   |                                 
+                    | | +-------------------+                 | +-------------------+ |                                 
+                    | | |                 | |<--------------->| |                 | | |                                 
+                    | +-|-----------------+ |                 +-|-----------------+ | |                                 
+                    |   +-------------------+                   +-------------------+ |                                 
+                    +-------|----------------------------------------------|----------+                                 
+                            |                                              |                                            
+                            |     <private cloud>  |-----------------------|----------------------
+                            |                      |  <public cloud>       |                                            
+                   watch,create,update,delete...   |         watch,create,update,delete...                               
+                            |                      |                       |                                            
+                    +-------v-----------+          |           +-----------v-----------+                                 
+                    |hub                |          |           |managed cluster        |                                 
+                    |                   |     logic|ally       |                       |                                 
+                    | +--------+        |<- - - - -|- - - - - >| +--------+            |                                 
+                    | | CRDs...|        |          |           | |CRDs... |            |                                 
+                    | +--------+        |          |           | +--------+            |                                 
+                    +-------------------+          |           +-----------------------+     
+```
+
+A real use case would be: The hub cluster is installed in a **private network** while the managed cluster is in a public cloud. right now, the managed cluster cannot connect to the hub cluster. **It also means deployments on managed cluster can not apply any resources to the hub cluster directly**.
```

Comment:
@clyang82 @qiujian16   Hello, we have added a real use case here, please have a review and see whether it's clear for you. Thanks~

Reply:
quick question, in the above case. Is there any reason that the cluster-manager controllers need to be in the management cluster?

Reply:
That's a good point! The controller could be running in the hub cluster too in the above case.
In practice, I got a case that the management cluster is super powerful, and the customer wants to leverage it for serval hub clusters, therefore hub clusters could be very light-weigh.